README.md

```diff
@@ -235,10 +260,27 @@ Variables in vars.yml
       - `name`: Name of the custom file output element.
       - `type`: Type of the output element.
       - `custom_config_files`: List of custom configuration files are deployed to /etc/rsyslog.d. [ '/path/to/custom_A.conf', '/path/to/custom_B.conf' ]. Default to none.
+   -  ** `type: remote_files`**
+      - `facility`: facility; default to `*`
+      - `severity`: severity; default to `*`
+      - `exclude`: exclude; default to none.
+      - `remote_log_path`: full path to store the filtered logs.
+      - `remote_sub_path`: relative path to logging_system_log_dir to store the filtered logs.
+	                       if `remote_log_path` nor `remote_sub_path` are not specified, the remote_file output configured with the default settings.
 - `logging_inputs` : List of optional logs collections, dictionaries with `name`, `type` and `state` attributes, that were pre-configured.
         - `name`: Unique name of the input.
-          `type`: The type of the pre-configured logs to collect. **Note:** Currently ['viaq', 'viaq-k8s', 'ovirt'] are supported for the elasticsearch output.
+          `type`: The type of the log inputs. **Note:** Currently [`basics`, `files`, `ovirt`, `remote`, and `viaq`] are supported.
           `state`: The state of the configuration files states if they should be `present` or `absent`. Default to `present`.
+   -  ** `type: basics`**
+      - `rsyslog_imjournal_ratelimit_burst`: set to imjournal RateLimit.Burst. Default to 20000.
+      - `rsyslog_imjournal_ratelimit_interval`: set to imjournal RateLimit.Interval. Default to 600.
+      - `rsyslog_imjournal_persist_state_interval`: set to imjournal PersistStateInterval. Default to 10.
+   -  ** `type: files`**
+      - `rsyslog_input_log_path`: File name to be read by the imfile plugin. The value should be full path. Wildcard '*' is allowed in the path.  Default to `/var/log/containers/*.log`.
+   -  ** `type: remote_files`**
+      - `udp_port`: if the port number is given, rsyslog is configured to listen at the udp port number. Default to 514.
+      - `tcp_port`: if the port number is given, rsyslog is configured to listen at the tcp port number. Default to 514.
+      - `tcp_tls_port`: if the port number is given, rsyslog is configured to listen at the tls tcp port number. Default to 6514.
```

Comment:
this needs certs, key?

Reply:
Right. The tls part hasn't been touched yet. :(